packages/lit-dev-content/site/tutorials/content/tooltip/07.md

```diff
@@ -0,0 +1,58 @@
+You've polished the tooltip's behavior with animations. Now it's time to ensure
+great performance. Since a tooltip displays only when the user interacts with its
+target, it doesn't need to be created until then. If a page uses a lot of
```

Comment:
Using callbacks to previous parts of a sentence like `then` in this case may lead to readers getting confused or having to double back if they're skimming. I recommend changing this to something like

> Since a tooltip displays only when a user interacts with the target, the tooltip does not need to be created until interaction.

Reply:
Done